### PR TITLE
Added Active class in sidebar

### PR DESCRIPTION
**Pull request by varun1233**

### CODE REVIEW

File: components/utility.css [hunk 1]
```diff
@@ -88,3 +88,11 @@ h6 {
 .text-align-right {
   text-align: right;
 }
+.selected-component {
+  color: var(--primary-color) !important;
+}
+.selected-component:hover {
+  color: var(--secondary-font-color) !important;
```
Comment (mehulsatardekar) [inline components/utility.css]: avoid the use of !important instead what u can do add specificity here

**Review by mehulsatardekar — APPROVED** [review]
rest part looks good to me.

File: components/utility.css (hunk 1 above)
Comment (varun1233) [reply components/utility.css]: Added for this specific one only because I wanted this class to override others.

Comment (prathameshpcodes) [reply components/utility.css]: Yeah, avoid using !important otherwise everything is good for merge

**Review by prathameshpcodes — APPROVED** [review]

File: components/utility.css (hunk 1 above)
Comment (varun1233) [reply components/utility.css]: Rectified it

**Pull request merged**